src/Json.Schema.ToDotNet/PropertyInfoDictionary.cs

```diff
@@ -654,5 +668,50 @@ private TypeSyntax GetConcreteType(string propertyName, string interfaceName)
 
             return type;
         }
+
+        internal TypeSyntax MakeProperIntegerType(GenerateJsonIntegerOption generateJsonIntegerAs,
```

Comment:
>MakeProperIntegerType [](http://example.com/codeflow?start=28&length=21)

I'm not entirely sure about this approach. Let me think about it. :) Let's get this change in, then work on guid & number and I will discuss this further with you.

Reply:
Ok. Looking at UUID PR now. Thanks.